probe-rs-target/Cargo.toml

```diff
@@ -12,6 +12,13 @@ keywords = ["embedded"]
 license.workspace = true
 
 [dependencies]
+base64 = "0.21.7"
+displaydoc = "0.2"
```

Comment:
Do we really want this? Convenient when it works, sure, but extremely inconsistent when you can't use it. Also pretty much unmaintained, contributing into the project is a chore.

Reply:
I like it because unifies doc comments and the error message and makes it impossible to lose sync and avoids writing it twice.
Sure it's not actively seeing changes, but I think the maintainer is still active.

Reply:
[FYI](https://github.com/yaahc/displaydoc/pull/45)